### ISSUE TITLE
SMS with v6.26.4 not working

### ISSUE THREAD

**Issue by Saaaam** [alexlance/signal-android-classic#8]
Hello :)

First of all, thank you for you great work on this fork. I read on the other topic/issue that SMS were working for you with v6.26.4, I just installed it on a new phone from the release page (universal apk), and sadly, I am not able to send SMS. When I try to send a message to someone who doesn't have Signal, this message is displayed on the screen : "The user doesn't use Signal".

Should I try a previous version, or modify something to the v6.26.4 to make SMS work?

Thank you for your help  ^^

**Comment by WeareTheDraiken** [comment]
Keep your Hand on the send button to change it from Signal to SMS.

Also, unfortunately, if you check the other issue thread, Signal started removing the code for SMS entirely, which became an Issue in the latest version of Signal being forked.

So Alexlance hasn't been updating the app,  so you won't really be able to really update the fork much.

Though, mind you , it's not an impossible issue to sort , it's just that it needs an idept programmer with free time who could put the code back in an updated fork.

That's why he left this fork project still open, in case someone with enough free time decides to take it over.

**Comment by Saaaam** [comment]
Yep, keeping my hand on the Send button is actually what I do (like I always used to  ;)  )

But it now displays a box with the options "Tonight at 9pm / Tomorrow at 8am / Tomorrow at 12am / Select a date and time"... so it became just a button to delay the sending of message

**Comment by WeareTheDraiken** [comment]
If I remember correctly you have to set signal as your sms messenger (it's what I did) , so I don't know which phone brand you use but this is normally done in settings under default appa.

**Comment by Saaaam** [comment]
It is a Redmi Note 12. And apparently, I indeed remember that I had to set Signal as my defaut sms app the first time I did that some months ago (https://www.01net.com/astuces/comment-configurer-signal-comme-application-sms-par-defaut-sur-android-2029975.html), but I do not have this option, either from Siignal, or the system... :/

**Comment by WeareTheDraiken** [comment]
Don't give up. There's still another way to do it.

If you install any other another app that is used for SMS then when you open the app, it will ask you to set SMS as default which will open the settings but list every app on your phone available for sms.

Then select signal.

After that, you can install the other app.

**Comment by Saaaam** [comment]
Sorry, I didn't take the time to answer, but your technique worked.... Thanks a lot :)
